README.md

```diff
@@ -12,9 +12,12 @@ This repository contains `scalajs-env-phantomjs` for Scala.js 1.x. In Scala.js
 // project/plugins.sbt
 addSbtPlugin("org.scala-js" % "sbt-scalajs-env-phantomjs" % "1.0.0")
 
+// Add an explicit import if using project/*.scala
+import org.scalajs.jsenv.phantomjs.sbtplugin.PhantomJSEnvPlugin.autoImport._
+
 // build.sbt (inside .settings(...) for multi-project builds)
-jsEnv := PhantomJSEnv().value
-scalaJSLinkerConfig ~= { _.withESFeatures(_.withUseECMAScript2015(false)) }
+Test / jsEnv := PhantomJSEnv().value
+Test / scalaJSLinkerConfig ~= { _.withESFeatures(_.withUseECMAScript2015(false)) }
```

Comment:
Wrt the `jsEnv`, no issue; just limiting it's scope. The `scalaJSLinkerConfig` OTOH is critical that I scope by `Test` because I don't the real/main/non-test output coming out in ES3. I want to keep the main artifacts on ES6 and only limit the test JS to ES3 for PhantomJS. I could be wrong but I *think* the 99% case is going to be running tests cos it's 1 or 2 orders of magnitude faster than Node.